cli/internal/dataplane/write.go

```diff
@@ -28,6 +28,7 @@ import (
 const (
 	DefaultWriteDop              = 16
 	DefaultBlockSize             = 4 * 1024 * 1024
+	DefaultTimeWindow			 = 0 // seconds
```

Comment:
hm, not sure why github is showing this as misaligned - looks aligned in vscode on my end. will see if i have a spaces/tabing config issue